Lib/socket.py

```diff
@@ -348,6 +349,8 @@ def makefile(self, mode="r", buffering=None, *,
     if hasattr(os, 'sendfile'):
 
         def _sendfile_use_sendfile(self, file, offset=0, count=None):
+            import selectors
```

Comment:
Let's leave a comment to explain why so it's not "tidied up" later:
```suggestion
            # Lazy import to improve module import time
            import selectors
```